static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -68,11 +80,57 @@ export function GithubInstallationSelect({installation_info}: GithubInstallation
             />
           )}
           <span>{`${installation.github_account}`}</span>
+          {installation.installation_id === '-1' || has_scm_multi_org || isSelfHosted ? (
```

Comment:
Bundling some of these checks into one variable might help readability and debugging:
```ts
const requiresUpgrade  = !(installation.installation_id === '-1' || has_scm_multi_org || isSelfHosted)
```

Reply:
Also is it possible for the `installation.installation_id` to be `-1`? I kinda figured the `-1` was only the `installationID` state value.